frontend/app/src/composables/filter-paginate.ts

```diff
@@ -218,7 +218,7 @@ export const usePaginationFilters = <
         }
 
         logger.error(error);
-        if (code === '500') {
+        if (code && ['400', '500'].includes(code)) {
```

Comment:
This is not exactly what I meant though, this will only show the error for 400 and 500 explicitly. I had in mind more of https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses so any 4xx or 5xx error code